.gitignore

```diff
@@ -9,4 +9,5 @@ Gemfile.lock
 Vagrantfile
 coverage/
 docs/build/
-.yardoc/
\ No newline at end of file
+.yardoc/
+tmp/
```

Comment:
Added tmp/local_secret.txt to .gitignore because it's generated when running tests.